pkg/services/sqlstore/migrator/dialect.go

```diff
@@ -209,6 +211,11 @@ func (b *BaseDialect) RenameTable(oldName string, newName string) string {
 	return fmt.Sprintf("ALTER TABLE %s RENAME TO %s", quote(oldName), quote(newName))
 }
 
+func (b *BaseDialect) RenameColumn(tableName, oldName, newName string) string {
+	quote := b.dialect.Quote
+	return fmt.Sprintf("ALTER TABLE %s RENAME COLUMN %s TO %s", quote(tableName), quote(oldName), quote(newName))
```

Comment:
Support for this was introduced since:

- **SQLite v3.25:** released at _2018-09-15_, which I guess is acceptable 🤔 
- **MySQL v8:** released at _2020-01-13_, which I guess is a bit more risky 🤔
- **PostgreSQL:** very looong time ago, if I'm not wrong ✅ 

Therefore, the alternative approach I suggest is to add specific support for the MySQL engine [through `CHANGE` statement](https://phoenixnap.com/kb/how-to-rename-column-mysql). What do you think?

For SQLite I guess there's no alternative, but I guess it's acceptable also considering SQLite it's not supported for production environments.

Bonus: If we introduce this for Grafana v9.0, then it's even more reasonable, I think.

Reply:
As for MySQL I agree with you - I'd use `change` to make sure older MySQL versions are supported as well (not sure how common it is).

Reply:
Yeah, I finally had to do it because otherwise our integration pipelines were failing. See [2784f8](https://github.com/grafana/grafana/pull/47765/commits/2784f84361ce9a545809b61f513b9e68924d4f66) for more details.

I guess the versions we use for these tests are the ones we consider the minimum supported versions, so I guess it's good enough. Anyway, thanks for sharing your thoughts on this! :)